apps/frontend/src/renderer/components/settings/DisplaySettings.tsx

```
@@ -110,16 +154,28 @@ export function DisplaySettings({ settings, onSettingsChange }: DisplaySettingsP
-            <ZoomOut className="h-4 w-4 text-muted-foreground shrink-0" />
+            <button
+              onClick={handleZoomOut}
+              disabled={displayScale <= UI_SCALE_MIN}
+              className={cn(
+                'p-1 rounded-md transition-colors shrink-0',
+                'hover:bg-accent text-muted-foreground hover:text-foreground',
+                'focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring',
+                'disabled:opacity-50 disabled:cursor-not-allowed disabled:hover:bg-transparent'
+              )}
+              title={`Decrease scale by ${UI_SCALE_STEP}%`}
+            >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)